docs/csharp/whats-new/tutorials/static-abstract-interface-methods.md

```diff
@@ -0,0 +1,147 @@
+
+:::code language="csharp" source="./snippets/staticinterfaces/Utilities.cs" id="MidPoint":::
+
+Any type that implements the `INumber` interface must include a definition for `operator +`, and `operator /`. The denominator is defined by `(T.One + T.One)` to create the value `2` for any numeric type. The readonly properties `Zero` and `One` are also implemented by every type that implements `INumber`. Using that property rather than `2` forces the denominator to be the same type as the two parameters.
+
+You define static abstract members in an interface using familiar syntax: You add the `static` and `abstract` modifiers to any static member that doesn't provide an implementation. The following example defines an `IBoolean<T>` interface that can be applied to any type that overrides `operator true` and `operator false`:
+
+:::code language="csharp" source="./snippets/staticinterfaces/GetNext.cs":::
+
+The constraint that the type argument, `T` implements `IGetNext<T>` ensures that the signature for the operator includes the containing type, or its type argument. Many operators enforce that its parameters must match the type, or be the type parameter constrained to implement the containing type. Without this constraint, the `++` operator couldn't be defined in the `IGetNext<T>` interface.
+
+You can create a structure that creates a string of 'A' characters where each increment adds another character to the string using the following code:
+
+:::code language="csharp" source="./snippets/staticinterfaces/RepeatSequence.cs":::
+
+More generally, you can build any algorithm where you might want to define `++` to mean "produce the next value of this type". Using this interface produces clear code and results:
+
+:::code language="csharp" source="./snippets/staticinterfaces/Program.cs" id="TestRepeat":::
+
+The preceding example produces the following output:
+
+```powershell
+A
+AA
+AAA
+AAAA
+AAAAA
+AAAAAA
+AAAAAAA
+AAAAAAAA
+AAAAAAAAA
+AAAAAAAAAA
+```
+
+This small example demonstrates the motivation for this feature. You can use natural syntax for operators, constant values, and other static operations. You can explore these techniques when you create multiple types that rely on static members, including overloaded operators. Define the interfaces that match your types' capabilities and then declare those types' support for the new interface.
+
+## Generic math and experimental features
+
+The motivating scenario for allowing static methods, including operators, in interfaces is to support *generic math* algorithms. The [System.Runtime.Experimental](https://www.nuget.org/packages/System.Runtime.Experimental/) NuGet package contains interface definitions for many arithmetic operators, and derived interfaces that combine many arithmetic operators in an `INumber<T>` interface. Let's apply those types to build a `Point<T>` class that can use any numeric type for `T`. The point can be moved by some `XOffset` and `YOffset` using the `+` operator.
```

Comment:
```suggestion
The motivating scenario for allowing static methods, including operators, in interfaces is to support *generic math* algorithms. The [System.Runtime.Experimental](https://www.nuget.org/packages/System.Runtime.Experimental/) NuGet package contains interface definitions for many arithmetic operators, and derived interfaces that combine many arithmetic operators in an `INumber<T>` interface. Let's apply those types to build a `Point<T>` record that can use any numeric type for `T`. The point can be moved by some `XOffset` and `YOffset` using the `+` operator.
```
?
I'm fine with class too (as in IL it's a class anyway), but why don't type it as _record_ when it's declared that way in the code?